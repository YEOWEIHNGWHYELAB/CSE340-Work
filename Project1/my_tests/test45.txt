t1 ((a).(b))|(c) , t2 (b).(c), t3 c #
" ab c ab
c ab c "
