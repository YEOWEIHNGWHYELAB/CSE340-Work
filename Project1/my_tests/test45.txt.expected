t1 , "ab"
t1 , "c"
t1 , "ab"
ERROR